WPMedia/engine-theme-sdk
# THEMES-1072: Remove LazyLoad from this repo

## Description

This PR removes the LazyLoad component from this repo, as it is now it `arc-themes-components`.

## Jira Ticket

- [THEMES-1072](https://arcpublishing.atlassian.net/browse/THEMES-1072)

## Acceptance Criteria

- updating import references

- move mocks from tests (if exist)

## Test Steps

- See `arc-themes-blocks` PR: https://github.com/WPMedia/arc-themes-blocks/pull/1644

## Dependencies or Side Effects

- `arc-themes-blocks` PR: https://github.com/WPMedia/arc-themes-blocks/pull/1644
- `arc-themes-components` PR: https://github.com/WPMedia/arc-themes-components/pull/173

## Review Checklist

        please explain why (so that we can fix it whenever it gets refactored).


[THEMES-1072]: https://arcpublishing.atlassian.net/browse/THEMES-1072?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
